Description: Don't run /etc/grub.d/README, in case it ended up executable
Author: Colin Watson <cjwatson@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/537123
Forwarded: no
Last-Update: 2010-03-22

Index: b/util/grub-mkconfig_lib.in
===================================================================
--- a/util/grub-mkconfig_lib.in
+++ b/util/grub-mkconfig_lib.in
@@ -159,6 +159,7 @@
   if test -f "$1" ; then
     case "$1" in
       *.dpkg-*) return 1 ;; # debian dpkg
+      README*)  return 1 ;; # documentation
     esac
   else
     return 1
